Articles/Blog/2021-03-useful-practices-for-SEoMsDSP.md

```diff
@@ -168,7 +158,7 @@ understanding and addressing the unique challenges in leading multi-institutiona
 dispersed scientific software development teams while still producing effective and usable software,
 particularly teams composed of members from scientific but not necessarily software engineering
 backgrounds.  He has worked in industry as a software engineer and release manager and has been at LBNL for  the past 20
-years.  He also tortures local music venue attendees playing bass in rock bands.
+years.  He also tours local music venue attendees playing bass in rock bands.
```

Comment:
Maybe this is too much of my odd sense of humor?  Would "torments" be better?

Reply:
> Maybe this is too much of my odd sense of humor? Would "torments" be better?

:)) Either is ok.